shared/bundle_analysis/parser.py

```diff
@@ -215,30 +243,49 @@ def _parse_modules_event(self, prefix: str, event: str, value: str):
     def _create_associations(self):
         # associate chunks to assets
         inserts = []
-        for chunk_id, asset_names in self.chunk_asset_names_index.items():
-            assets = self.db_session.query(Asset).filter(
-                Asset.session == self.session,
-                Asset.name.in_(asset_names),
-            )
+        assets: list[Asset] = self.db_session.query(Asset).filter(
+            Asset.session_id == self.session.id,
+        ).all()
+
+        asset_name_to_id = {
+            asset.name: asset.id for asset in assets
```

Comment:
Is it possible to have multiple assets with the same name? Would that overwrite some IDs?

Reply:
it is not possible, but if it was, you're correct that this solution would be broken

Reply:
It would be helpful to add a comment stating that